lib/stimulus_reflex/broadcasters/selector_broadcaster.rb

```diff
@@ -64,5 +58,26 @@ def selector?
     def to_s
       "Selector"
     end
+
+    private
+
+    def create_update_collection(selectors, html)
+      updates = selectors.is_a?(Hash) ? selectors : {selectors => html}
+      updates.map do |key, value|
+        Update.new(key, value, reflex)
+      end
+    end
+
+    StimulusReflex::SelectorBroadcaster::Update = Struct.new(:key, :value, :reflex) do
```

Comment:
We could omit the fully qualified namespace on the `Update` struct, but I suppose it helps grokkability. 

Also, considering that we've added some methods/behavior to `Update`, I propose that it deserves a separate file and actual class definition.

Reply:
Updated, but we still need CR 5.0.0.pre2 for the tests to pass 👍